### ISSUE TITLE
Directory Exploration Commands Break the Minimap 

### ISSUE THREAD

**Issue by K4LCIFER** [wfxr/minimap.vim#71]
## Check list

- [X] I have read through the [README](https://github.com/wfxr/minimap.vim/blob/master/README.md) (especially F.A.Q section)
- [X] I have searched through the existing issues

## Environment info

- OS
    - [x] Linux
    - [ ] Mac OS X
    - [ ] Windows
    - [ ] Others:

## Version info

NVIM v0.5.0-dev
NVIM-qt v0.2.16.0

## Question / Problem and steps to reproduce
Directory exploration commands (`:Explore`, `:Sexplore`, `:Vexplore`, etc) break the minimap. 

Steps:
1. Open a file.
2. run a directory exploration command and open a file.
3. run a help command and observe that the minimap all of a sudden enlarges to half the screen and cannot be fixed unless you restart vim.

**Comment by rabirabirara** [comment]
I can't seem to recreate this issue.  Could you possibly record a GIF of your screen when this happens?  Or, is it opening the help window that's the problem?  If that's so, then you can add the `help` filetype to the option `g:minimap_close_filetypes`, which should prevent the minimap from opening on the help window.

**Comment by streetmask** [comment]
works like this here.
![gp8m6bFyDx](https://user-images.githubusercontent.com/30765414/117564461-bc1a4a80-b0de-11eb-969a-420dca44acd8.gif)
but if I use nvim-tree instead of rnvimr to open file, It becomes normal.

**Comment by ZNielsen** [comment]
It looks like there's some window assumptions happening when that new file is being opened, probably in the function from rnvimr. My guess is the commands are sent to:
1) Open a new file in a vertical split
2) Move the cursor to the rightmost window, which is assumed to be the newly opened window but is actually the minimap.
3) Resize the window the cursor is in to be half the size of the window the command started in. This results in the new file being the size that the minimap was, but sandwiched in the middle. I didn't break out an image editor to check, but eyeballing it it looks just about right.

Can you try this again, but with any window that you manually opened on the right?

Long story short, I don't think this is a problem with minimap.vim.

**Comment by streetmask** [comment]
@ZNielsen you are right. It seems to be a compitability problem of rnvimr. Either manually or using other plugins, this works perfect.
